website/static/js/registrationUtils.js

```diff
@@ -303,18 +303,26 @@ var RegistrationEditor = function(urls, editorId) {
         return schema.pages;
     });
 
-    self.lastSaveTime = ko.computed(function() {        
+    self.lastSaveTime = ko.computed(function() {
         if(!self.draft()) {
             return null;
         }
         return self.draft().updated;
     });
     self.formattedDate = formattedDate;
-    
+
     self.iterObject = $osf.iterObject;
 
     self.extensions = {};
 };
+
+// View model for binding to templates
+var Submission = function() {
```

Comment:
Please delete this if it's unused. 

We can pass an empty object to ko.renderTemplate.